vendor/install-getowner-polyfill.js

```diff
@@ -153,19 +158,38 @@
       return container.lookupFactory.apply(container, arguments);
     };
 
+    function getOwner(object) {
+      if (object[OWNER]) {
+        return object[OWNER];
+      }
+
+      // Fallback to finding the owner on the container
+      var container = object.container;
+      if (!container) { return; }
+
+      if (!container[OWNER]) {
+        var owner = new FakeOwner(object);
+        container[OWNER] = owner;
+      }
+
+      return container[OWNER];
+    }
+
+    function setOwner(object, owner) {
+      object[OWNER] = owner;
+    }
```

Comment:
This probably belongs in the other branch, although it may not matter with hoisting rules...